src/display/Table/Table.tsx

```diff
@@ -0,0 +1,168 @@
+  min-height: ${sizing(100)};
+  padding: ${sizing(4)};
+  outline: none;
+  align-self: start;
+  word-break: break-word;
+  forced-color-adjust: none;
+  border-spacing: 0;
+
+  &[data-focus-visible] {
+    outline: 2px solid ${theme.color.primary};
+    outline-offset: -1px;
+  }
+`;
+
+const StyledTableBody = styled(TableBody)<{
+  evenRowsBackgroundIsColored?: boolean;
+}>`
+  ${({ evenRowsBackgroundIsColored }) => css`
+    & tr:nth-child(even) {
+      background-color: ${evenRowsBackgroundIsColored && theme.color.gray100};
+
+      &[data-selected] {
+        background: ${evenRowsBackgroundIsColored && theme.color.primary200};
+      }
+    }
+  `}
+`;
+
+const StyledColumnContainer = styled.div`
+  position: relative;
+  z-index: 1;
+`;
+
+const StyledColumnResizer = styled(ColumnResizer)`
+  position: absolute;
+  top: ${sizing(-6)};
+  right: ${sizing(-8)};
+  width: ${sizing(6)};
+  background-color: ${theme.color.gray};
+  height: ${sizing(25)};
+  flex: 0 0 auto;
+  touch-action: none;
+  box-sizing: border-box;
+  border: ${sizing(2)};
+  border-style: none solid;
+  border-color: transparent;
+  background-clip: content-box;
+  z-index: 2;
+
+  &[data-resizable-direction="both"] {
+    cursor: ew-resize;
+  }
+
+  &[data-resizable-direction="left"] {
+    cursor: e-resize;
+  }
+
+  &[data-resizable-direction="right"] {
+    cursor: w-resize;
+  }
+
+  &[data-focus-visible] {
+    background-color: ${theme.color.primary};
+  }
+
+  &[data-resizing] {
+    border-color: ${theme.color.primary};
+    background-color: transparent;
+  }
+`;
+
+const Table = <T,>(props: TableProps<T>) => {
+  return (
+    <StyledResizableTableContainer>
+      <StyledTable
+        {...props}
+        aria-label={props.tableTitle}
+        selectionMode={props.selectionMode || "multiple"}
+        defaultSelectedKeys={props.defaultSelectedKeys}
+      >
+        <TableHeader columns={props.dataColumns}>
+          {props.dataColumns.map((column) => (
+            <TableColumn key={column.id} isRowHeader>
+              <StyledColumnContainer>
+                {column.name}
+                {column.isResizable && (
+                  <StyledColumnResizer
+                    aria-label={`Resize ${column.textValue}`}
+                  />
+                )}
+              </StyledColumnContainer>
+            </TableColumn>
+          ))}
+        </TableHeader>
+        <StyledTableBody
+          items={props.dataRows}
+          evenRowsBackgroundIsColored={props.evenRowsBackgroundIsColored}
+        >
+          {props.dataRows.map((row) => (
+            <TableRow
+              columns={props.dataColumns}
+              key={row.id}
+              isDisabled={row.isDisabled}
+              data-is-colored={row.isColored}
+            >
+              {props.dataColumns.map((column) => {
+                return (
+                  <TableCell key={column.id}>
+                    {(row as Record<string, any>)[column.id ?? "name"]}
```

Comment:
maybe we can find better than **any** here

Reply:
Record need two arguments, the first one is what we want and the second one is what is it.